deeppavlov/skills/intent_matching_skill/faq_skill.py

```diff
@@ -0,0 +1,82 @@
+from typing import Tuple, Optional
+
+from deeppavlov import train_model
+from deeppavlov import build_model
+from deeppavlov.core.skill.skill import Skill
+from deeppavlov.core.common.file import read_json
+from deeppavlov.core.common.file import find_config
+from deeppavlov.core.data.utils import update_dict_recursive
+
+
+class IntentMatchingSkill(Skill):
+    """Skill, matches utterances to intents, returns predefined answers.
+
+    Allows to create skills that give answers to corresponding intents
+    Skill returns response and confidence.
+
+    Args:
+        data_path: URL or local path to '.csv' file that contains two columns with Intents and Answers.
+            User's utterance will be compared with Intents column and respond will be selected
+            from matching row from Answers column.
+        x_col_name: Name of the column in '.csv' file, that represents Intents column.
+        y_col_name: Name of the column in '.csv' file, that represents Answer column.
+        save_path: Path, where models will be saved
+        load_path: Path, where models will be loaded from
+        edit_dict: Dictionary of edits in config
+
+    Attributes:
+        model: Classifies user's questions
+    """
+
+    def __init__(self, data_path: str = None, x_col_name: str = None, y_col_name: str = None, edit_dict: dict = None,
+                 save_path: str = None, load_path: str = None) -> None:
+        if load_path is not None and \
+                (save_path is not None or data_path is not None or x_col_name is not None or y_col_name is not None):
+            raise ValueError("If you specify 'load_path', you can't specify anything else, "
+                             "because it leads to ambiguity")
+
+        model_config = read_json(find_config('tfidf_autofaq'))
```

Comment:
@velik97 So, what do you decide?

Reply:
Yes, we should be able to use other configs. But none of the other configs  has proper output, which should be “answer+score”. This might take a lot of time, because each config has different reason for not heaving such output. Thus, I’d like to work on this in next PR.